### PR TITLE
[stable10] Backport of languages from master

### PR DESCRIPTION
**Pull request by DeepDiver1975**

### CODE REVIEW

File: core/l10n/de.json [hunk 1]
```diff
@@ -368,6 +368,7 @@
     "Personal note from the sender: %s." : "Persönliche Notiz des Absenders: 1%s .",
     "Automatic updating is disabled in config.php. To upgrade your instance, please use the command line updater (occ upgrade)." : "Automatische Aktualisierung ist in der Konfiguration deaktiviert. Zur Aktualisierung der Instanz bitte die Kommandozeile benutzen (php occ upgrade).",
     "Imprint" : "Impressum",
-    "E-Mail sent" : "E-Mail gesendet"
+    "E-Mail sent" : "E-Mail gesendet",
+    "Privacy Policy" : "Datenschutzerkärung"
```
Comment (PVince81) [inline core/l10n/de.json]: Typo "erkärung"

File: settings/l10n/de_DE.js [hunk 2]
```diff
@@ -362,6 +362,17 @@ OC.L10N.register(
     "You are not a member of any groups." : "Du bist kein Mitglied einer Gruppe.",
     "In a clustered setup please make sure to sync the config.php file across all nodes." : "In einem gruppierten Umgebung stellen Sie bitte sicher, dass die config.php-Datei über alle Knoten synchronisiert wird.",
     "You are using %s" : "Sie verwenden %s",
-    "These are the web, desktop and mobile clients currently logged in to your %s." : "Diese Web-, Desktop- und Mobil-Clients sind aktuell an 1%s angemeldet"
+    "These are the web, desktop and mobile clients currently logged in to your %s." : "Diese Web-, Desktop- und Mobil-Clients sind aktuell an 1%s angemeldet",
```
Comment (PVince81) [inline settings/l10n/de_DE.js]: is the syntax `1%s` correct here despite it not being in the original message ? I see it used in other older messages

**Comment by PVince81** [comment]
@DeepDiver1975 added comments, we should fix this in Transifex upstream.

Merging the PR and let's process the typo upstream and in a subsequent PR

**Review by PVince81 — APPROVED** [review]

File: settings/l10n/de_DE.js (hunk 2 above)
Comment (DeepDiver1975) [reply settings/l10n/de_DE.js]: transifex is changing this that way ..... strange

Comment (DeepDiver1975) [reply settings/l10n/de_DE.js]: there is not much I can do :-(

**Pull request merged**